### PR TITLE
refactor(assets/respec2): wavy by default

### PR DESCRIPTION
**Pull request by saschanaz**
Solving problem raised in https://github.com/w3c/respec/pull/2416#issuecomment-510440998.

### CODE REVIEW

File: assets/respec2.css [hunk 1]
```diff
@@ -56,24 +56,17 @@ a.bibref {
...
-  display: inline-block;
-  position: relative;
-  /* Red squiggly line */
-  background: url(data:image/gif;base64,R0lGODdhBAADAPEAANv///8AAP///wAAACwAAAAABAADAEACBZQjmIAFADs=)
-    bottom repeat-x;
+  text-decoration: red wavy underline;
 }
-@supports (text-decoration-style: wavy) {
+@supports not (text-decoration: red wavy underline) {
+  .respec-offending-element:not(pre) {
```
Comment (sidvishnoi) [inline assets/respec2.css]: Just to confirm, we're not adding `.respec-offending-element` to any `section`, right?

Comment (saschanaz) [reply assets/respec2.css]: I'm not sure, but haven't seen such case.

Comment (saschanaz) [reply assets/respec2.css]: @marcoscaceres Thoughts?

Comment (marcoscaceres) [reply assets/respec2.css]: I’m 99.9% sure we are not.

**Pull request merged**